src/app/Core/ExportServices/objectsToZIP.service.ts

```diff
@@ -15,7 +15,7 @@ interface ObjectToCsvBothProps {
     subFileNames: string[];
 }
 
-export function objectToZIP(props: ObjectToCsvBothProps): void {
+export function objectToZIP(props: ObjectToZIPProps): void {

```

Comment:
reject: this is no react component, so it has no props. Use options or so instead. The Props is a react only naming convention.

Reply:
Is "parameters" then the right name?

Reply:
There is no 'right' name. A good name would something describing the actual data of the parameter. Or just 'object' like the function name suggests. Perhaps the problem is the name of the function it says convert any object to zip, but that is not what it does.

Reply:
I tried my best. ;) 
Commit: "Rename "props" inside services "

Reply:
sorry, i have thought of the function parameters as you mentioned 'parameters'. But the types are actually meant to be called 'parameters' in your code. So 'parameters' is the right name :)
Because naming is so difficult you will often rename things multiple times as the code progresses. The only problem were the react reserved props keywords.